test/e2e/testing-manifests/statefulset/nginx/statefulset.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta2
+apiVersion: apps/v1
```

Comment:
LGTM as `StatefulSet in the apps/v1beta1 and apps/v1beta2 API versions is no longer served` according to https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/